modules/juce_audio_devices/native/juce_ios_Audio.cpp

```diff
@@ -1031,6 +1031,7 @@ struct iOSAudioIODevice::Pimpl      : public AsyncUpdater
         desc.componentFlags = 0;
         desc.componentFlagsMask = 0;
                
+        setAudioPreprocessingEnabled(true);

```

Comment:
Can/should we do this in an else block instead to avoid extra session mode swaps?

Reply:
Done